src/main.js

```diff
@@ -0,0 +1,327 @@
+//                 </p>
+//               </div>
+//             </li>
+//           </ul>
+
+//           <div class="film-details__new-comment">
+//             <div for="add-emoji" class="film-details__add-emoji-label"></div>
+
+//             <label class="film-details__comment-label">
+//               <textarea class="film-details__comment-input" placeholder="Select reaction below and write comment here" name="comment"></textarea>
+//             </label>
+
+//             <div class="film-details__emoji-list">
+//               <input class="film-details__emoji-item visually-hidden" name="comment-emoji" type="radio" id="emoji-smile" value="sleeping">
+//               <label class="film-details__emoji-label" for="emoji-smile">
+//                 <img src="./images/emoji/smile.png" width="30" height="30" alt="emoji">
+//               </label>
+
+//               <input class="film-details__emoji-item visually-hidden" name="comment-emoji" type="radio" id="emoji-sleeping" value="neutral-face">
+//               <label class="film-details__emoji-label" for="emoji-sleeping">
+//                 <img src="./images/emoji/sleeping.png" width="30" height="30" alt="emoji">
+//               </label>
+
+//               <input class="film-details__emoji-item visually-hidden" name="comment-emoji" type="radio" id="emoji-gpuke" value="grinning">
+//               <label class="film-details__emoji-label" for="emoji-gpuke">
+//                 <img src="./images/emoji/puke.png" width="30" height="30" alt="emoji">
+//               </label>
+
+//               <input class="film-details__emoji-item visually-hidden" name="comment-emoji" type="radio" id="emoji-angry" value="grinning">
+//               <label class="film-details__emoji-label" for="emoji-angry">
+//                 <img src="./images/emoji/angry.png" width="30" height="30" alt="emoji">
+//               </label>
+//             </div>
+//           </div>
+//         </section>
+//       </div>
+//     </form>
+//   </section>`
+// );
+
+
+//  ---------------------- ф-ция рендер -------------------
+const render = (container, template, place = `beforeend`) => {
+  if (container instanceof Element) {
+    container.insertAdjacentHTML(place, template);
+  }
+};
+
+
+//  ----------------------- containers -----------------------
+
+const siteHeaderElement = document.querySelector(`.header`);
+const siteMainElement = document.querySelector(`.main`);
+
+
+// --------------------- рендер на страницу -------------------
+render(siteHeaderElement, createProfileTemplate());
+render(siteMainElement, createMainNavigationTemplate());
+render(siteMainElement, createSortTemplate());
+render(siteMainElement, createFilmsListTemplate());
+
+
+// контейнер фильмов верхний
+const taskListElement = siteMainElement.querySelector(`.films-list__container`);
+
+// счётчик для фильмов сверху`
+new Array(TASK_COUNT)
+  .fill(``)
+  .forEach(
+      () => render(taskListElement, createFilmCardTaskTemplate())
```

Comment:
```suggestion
      .forEach(() => render(taskListElement, createFilmCardTaskTemplate()));
```
Так обычно не переносят. Если у тебя функция-стрелка в одну строку, зачем переносить?

Reply:
> Так обычно не переносят. Если у тебя функция-стрелка в одну строку, зачем переносить?

>>>не знаю.. так на первой лекции перенос делали на видео
Больше так не буду_)